### PR TITLE
Fixed an issue where it doesn't take *value* parameter.

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
`IgnoreFieldsFromJson` didn't take `value` parameter for its transformation even though it is a transformer. The PR fixed the issue.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by moishce — COMMENTED** [review]
I apologize for the delay, (it was marked as waiting for a reply by mistake)

File: Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/README.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ A transformer to remove selected fields from the JSON object.
 
 | **Argument Name** | **Description** |
 | --- | --- |
-| value | The value of the JSON object |
+| value | The primary value of the JSON object |
+| json_object | The value of the JSON object |
```
[the remark this answers is not in the document]
Comment (spearmin10) [reply Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/README.md]: @moishce OK, I've fixed it.

File: Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/IgnoreFieldsFromJson.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 args:
+- description: The primary value of the JSON object.
+  name: value
 - description: The value of the JSON object.
   name: json_object
```
[the remark this answers is not in the document]
Comment (spearmin10) [reply Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/IgnoreFieldsFromJson.yml:5]: @moishce Sure, I've added it.

[the remark this answers is not in the document]
Comment (spearmin10) [reply Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/IgnoreFieldsFromJson.yml:5]: @moishce It gots the error below at `
ci/circleci: Run Validations` by adding `hidden`. Is it OK?

```
[ERROR]: Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/IgnoreFieldsFromJson.yml: [ST111] - The field "hidden" in path 'args'-> 'json_object' was not defined in the scheme
To fix the problem, try running `demisto-sdk format -i Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/IgnoreFieldsFromJson.yml 
```

[the remark this answers is not in the document]
Comment (moishce) [reply Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/IgnoreFieldsFromJson.yml:5]: It's probably my mistake and it's not possible to make it hidden

[the remark this answers is not in the document]
Comment (spearmin10) [reply Packs/CommonScripts/Scripts/IgnoreFieldsFromJson/IgnoreFieldsFromJson.yml:5]: @moishce OK, I got it.

**Review by moishce — APPROVED** [review]

**Pull request merged**